docs/guide-release-process.md

```diff
@@ -319,13 +328,15 @@ git pull
 
 # Head to rn-tester package and update pods
 rm -rf packages/react-native-codegen/lib
-(cd packages/rn-tester && bundle exec pod update hermes-engine --no-repo-update)
+(cd packages/rn-tester && bundle install && bundle exec pod update hermes-engine --no-repo-update)
 
 # Commit only changes to packages/rn-tester/Podfile.lock
-git add packages/rn-tester/Podfile.lock
+git add ./Podfile.lock
```

Comment:
The old line is correct because we are running the above command as a subshell, so the cwd is still the `react-native` root.